aggregate.go

```diff
@@ -13,3 +13,11 @@ type aggregate struct {
 	interval      time.Duration
 	regexp        []*regexp.Regexp
 }
+
+func newAggregate(i time.Duration, res []*regexp.Regexp) *aggregate {
+	return &aggregate{
```

Comment:
I believe aggregate is **never** copied?
If it is copied anywhere (and not passed as a pointer), we need to make the mutex to a pointer and initialise it here.

Reply:
It's never copied.